source/Distribution.h

```diff
@@ -0,0 +1,41 @@
+/* Distribution.h
+Copyright (c) 2022 by DeBlister
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef DISTRIBUTION_H_
+#define DISTRIBUTION_H_
+
+#include "Angle.h"
```

Comment:
This can be a forward declaration (`class Angle;`).

Reply:
This is true but Distribution.cpp won't build unless Angle.h is included there.  Would a forward declaration in Distribution.h and full include in Distribution.cpp be preferable to how it is now?